src/main/groovy/com/stanfy/spoon/gradle/SpoonRunTask.groovy

```diff
@@ -114,7 +117,11 @@ class SpoonRunTask extends DefaultTask implements VerificationTask {
     if (testSize != TEST_SIZE_ALL) {
       // Will throw exception with informative message if provided size is illegal
       runBuilder.setTestSize(IRemoteAndroidTestRunner.TestSize.getTestSize(testSize))
-    }    
+    }  
+    
+    if (adbTimeout.toString().isInteger()){
```

Comment:
I do not fully understand what this statement should actually do... :) What is your intent? Do you want to make `adbTimeout` optional?